infrastructure/kctf/scripts/cluster/start.sh

```diff
@@ -18,6 +18,35 @@ gcloud compute routers nats create kctf-${CLUSTER_NAME}-nat-config --router-regi
 
 get_cluster_creds
 
+# GCSFUSE
+
+SUFFIX=$(echo "${PROJECT}-${CLUSTER_NAME}-${ZONE}" | sha1sum)
+BUCKET_NAME="kctf-gcsfuse-${SUFFIX:0:16}"
+GSA_NAME="${BUCKET_NAME}"
+GSA_EMAIL=$(gcloud iam service-accounts list --filter "name:${GSA_NAME}" --format 'get(email)' || true)
+if [ -z "${GSA_EMAIL}" ]; then
+  gcloud iam service-accounts create "${GSA_NAME}" --description "kCTF GCSFUSE service account ${CLUSTER_NAME} ${ZONE}" --display-name "kCTF GCSFUSE ${CLUSTER_NAME} ${ZONE}"
+  GSA_EMAIL=$(gcloud iam service-accounts list --filter "name:${GSA_NAME}" --format 'get(email)')
+fi
+if ! gsutil du "gs://${BUCKET_NAME}/"; then
+  gsutil mb -l eu "gs://${BUCKET_NAME}/"
+  gsutil uniformbucketlevelaccess set on "gs://${BUCKET_NAME}/"
+fi
+gcloud projects add-iam-policy-binding "${PROJECT}" --member "serviceAccount:${GSA_EMAIL}" --role roles/storage.objectAdmin
```

Comment:
what happens if the binding already exists?

Reply:
nothing, it just readds it